yarn.lock

```diff
@@ -1205,6 +1205,17 @@
   resolved "https://registry.yarnpkg.com/@rushstack/eslint-patch/-/eslint-patch-1.1.0.tgz#7f698254aadf921e48dda8c0a6b304026b8a9323"
   integrity sha512-JLo+Y592QzIE+q7Dl2pMUtt4q8SKYI5jDrZxrozEQxnGVOyYE+GWK9eLkwTaeN9DDctlaRAQ3TBmzZ1qdLE30A==
 
+"@teamsupercell/typings-for-css-modules-loader@^2.5.1":
```

Comment:
just wondering, is this supposed to be a dependency or dev dependency? it seems like it would only matter while building

Reply:
oop yes